### PR TITLE
dev/doc/univpoly.{txt => md}, split off primitive projection info

### PR DESCRIPTION
**Pull request by SkySkimmer**
Also delete useless `dev/doc/universes.txt`.

### CODE REVIEW

File: dev/doc/universes.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
-Notes on universe polymorphism and primitive projections, M. Sozeau
-===================================================================
+Notes on universe polymorphism
+------------------------------
 
 The new implementation of universe polymorphism and primitive
 projections introduces a few changes to the API of Coq. First and
```
Comment (Zimmi48) [inline dev/doc/universes.md]: Still mentions "primitive projection".

**Review by Zimmi48 — COMMENTED** [review]
Thank you for this.
Maybe you could add some verbatim quoting in the `primproj.md` file.

File: dev/doc/universes.md (hunk 1 above)
Comment (SkySkimmer) [reply dev/doc/universes.md]: Changed.

**Comment by SkySkimmer** [comment]
> Maybe you could add some verbatim quoting in the primproj.md file.

Changed.

**Review by Zimmi48 — APPROVED** [review]

**Pull request merged**